tests/Tests.FeatureManagement/FeatureFlags.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Tests.FeatureManagement
+{
+    static class FeatureFlags
```

Comment:
Why change Features to FeatureFlags?

Reply:
I can change it back to Features. In README, we use "FeatureFlag".